apps/console/src/features/core/api/user-store.ts

```diff
@@ -29,6 +29,35 @@ const httpClient = IdentityClient.getInstance()
     .httpRequest.bind(IdentityClient.getInstance())
     .bind(IdentityClient.getInstance());
 
+/**
+ * Gets details of the primary user store.
+ *
+ *
+ * @return {Promise<any>} response.
+ */
+export const getPrimaryUserStore = (): Promise<any> => {
```

Comment:
Is there a possibility to add the return type here?

Reply:
Fixed with commit 0a71bfe